staging/src/k8s.io/apiserver/pkg/util/flag/flags.go

```diff
@@ -43,9 +43,12 @@ func WarnWordSepNormalizeFunc(f *pflag.FlagSet, name string) pflag.NormalizedNam
 	return pflag.NormalizedName(name)
 }
 
-// InitFlags normalizes and parses the command line flags
+// InitFlags normalizes, parses, then logs the command line flags
 func InitFlags() {
 	pflag.CommandLine.SetNormalizeFunc(WordSepNormalizeFunc)
 	pflag.CommandLine.AddGoFlagSet(goflag.CommandLine)
 	pflag.Parse()
+	pflag.VisitAll(func(flag *pflag.Flag) {
+		glog.Infof("FLAG: --%s=%q", flag.Name, flag.Value)
```

Comment:
> Why wouldn't you know the flags you used to launch the process?

I'm not sure I truly understand the question, but in this case it's because the VM is long gone and all I have is a failed test and some logfiles, and so I want this in the log file.